### PR TITLE
remove certificate manager unused code

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What this PR does / why we need it**:
remove unused const
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/certificate/certificate_manager.go [hunk 1]
```diff
@@ -170,7 +166,8 @@ func (m *manager) Current() *tls.Certificate {
 // CertificateSigningRequestClient that has already been set. This method is to
 // support the one specific scenario where the CertificateSigningRequestClient
 // uses the CertificateManager.
-func (m *manager) SetCertificateSigningRequestClient(certSigningRequestClient certificatesclient.CertificateSigningRequestInterface) error {
+func (m *manager) SetCertificateSigningRequestClient(
+	certSigningRequestClient certificatesclient.CertificateSigningRequestInterface) error {
```
Comment (dixudx) [inline pkg/kubelet/certificate/certificate_manager.go]: Bad fmt. Please keep it as it is.

File: pkg/kubelet/certificate/certificate_manager.go [hunk 2]
```diff
@@ -357,7 +354,10 @@ func (m *manager) generateCSR() (csrPEM []byte, keyPEM []byte, err error) {
 // NOTE This is a copy of a function with the same name in
 // k8s.io/kubernetes/pkg/kubelet/util/csr/csr.go, changing only the package that
 // CertificateSigningRequestInterface and KeyUsage are imported from.
-func requestCertificate(client certificatesclient.CertificateSigningRequestInterface, csrData []byte, usages []certificates.KeyUsage) (certData []byte, err error) {
+func requestCertificate(
+	client certificatesclient.CertificateSigningRequestInterface,
```
Comment (dixudx) [inline pkg/kubelet/certificate/certificate_manager.go]: Bad fmt. As above.

File: pkg/kubelet/certificate/kubelet.go [hunk 3]
```diff
@@ -31,7 +31,13 @@ import (
 
 // NewKubeletServerCertificateManager creates a certificate manager for the kubelet when retrieving a server certificate
 // or returns an error.
-func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg *kubeletconfig.KubeletConfiguration, nodeName types.NodeName, ips []net.IP, hostnames []string, certDirectory string) (Manager, error) {
+func NewKubeletServerCertificateManager(
+	kubeClient clientset.Interface,
```
Comment (dixudx) [inline pkg/kubelet/certificate/kubelet.go]: Bad fmt. Please keep it as it is.

File: pkg/kubelet/certificate/kubelet.go [hunk 4]
```diff
@@ -81,7 +87,13 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 // client that can be used to sign new certificates (or rotate). It answers with
 // whatever certificate it is initialized with. If a CSR client is set later, it
 // may begin rotating/renewing the client cert
-func NewKubeletClientCertificateManager(certDirectory string, nodeName types.NodeName, certData []byte, keyData []byte, certFile string, keyFile string) (Manager, error) {
+func NewKubeletClientCertificateManager(
+	certDirectory string,
```
Comment (dixudx) [inline pkg/kubelet/certificate/kubelet.go]: Please keep it as it is.

**Review by dixudx — CHANGES_REQUESTED** [review]

File: pkg/kubelet/certificate/certificate_manager.go (hunk 1 above)
Comment (hzxuzhonghu) [reply pkg/kubelet/certificate/certificate_manager.go]: the line is too long, code convention does not prefer this

**Comment by hzxuzhonghu** [comment]
@dixudx keep the format as it is. Only remove unused const

**Comment by hzxuzhonghu** [comment]
@liggitt @mikedanese  Would you please take another look? Conflict too quickly.

**Pull request merged**